<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>618</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>618</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>618</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Seed Obfuscator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lnSeed">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>461</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblSeed">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Seed Phrase</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lnPassphrase">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>461</width>
      <height>28</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lblPassphrase">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Obfuscation Passphrase</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lnConfirmPassphrase">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>461</width>
      <height>28</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lblConfirmPassphrase">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirm Passphrase</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnObfuscate">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>461</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Obfuscate</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="grpResult">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>471</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Result</string>
    </property>
    <widget class="QTextEdit" name="txtResult">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>431</width>
       <height>151</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnDeobfuscate">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>461</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Debfuscate</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSeed_Obfuscator">
    <property name="title">
     <string>Seed Obfuscator</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuSeed_Obfuscator"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
